pages/instructorFileEditor/instructorFileEditor.ejs

```diff
@@ -87,28 +94,50 @@
+                                <% } %>
+                                <% if (!(Object.is(arr.length - 1, key))) { %>
+                                <span class="mx-2">/</span>
+                                <% } %>
+                              <% }); %>
+                            </span>
                         </div>
                         <div class="col-auto collapse <% if ((!fileEdit.failedPush) && (!fileEdit.alertChoice)) { %>show<% } %>" id="buttons-<%= fileEdit.uuid %>">
-                            <a class="btn btn-secondary" data-toggle="collapse" href="#help-<%= fileEdit.uuid %>" role="button" id="help-<%= fileEdit.uuid %>-button">Show help</a>
-                            <button id="file-editor-<%= fileEdit.uuid %>-save-button" name="__action" value="save_and_sync" class="btn btn-primary" disabled>Save and sync</button>
+                            <a id="help-<%= fileEdit.uuid %>-button"
+                               class="btn btn-light btn-sm"
+                               data-toggle="collapse" href="#help-<%= fileEdit.uuid %>"
+                               role="button" aria-expanded="false">
+                                <i class="far fa-question-circle" aria-hidden="true"></i>
+                                <span id="help-<%= fileEdit.uuid %>-button-label">Show help</span>
+                            </a>
+                            <button id="file-editor-<%= fileEdit.uuid %>-save-button" name="__action" value="save_and_sync" class="btn btn-light btn-sm" disabled>
+                                <i class="fas fa-save" aria-hidden="true"></i>
+                                Save and sync
+                            </button>
                         </div>
                     </div>
                 </div>
-                <div class="card-body row collapse" id="help-<%= fileEdit.uuid %>">
-                    <div class="container-fluid">
-                        <p>
-                            You are editing the file <code><%= fileEdit.fileNameForDisplay %></code>. To save changes, click <strong>Save and sync</strong> or use <strong>Ctrl-S</strong> (Windows/Linux) or <strong>Cmd-S</strong> (Mac).
-                            <% if (config.fileEditorUseGit) { %>
-                            Doing so will write your changes to disk, will push them to the remote GitHub repository, and will sync them to the database.
-                            <% } else { %>
-                            Doing so will write your changes to disk and will sync them to your local database. You will need to push these changes to the GitHub respository manually (i.e., not in PrairieLearn), if desired.
-                            <% } %>
-                            If you reload or navigate away from this page, any unsaved changes will be lost.
-                        </p>
+                <div class="collapse" id="help-<%= fileEdit.uuid %>">
+                    <div class="card-body">
```

Comment:
By moving the `card-body` class to a div nested within the `collapse` div, there won't be a jumpy animation as it opens and closes.